java/image.bzl

```diff
@@ -215,7 +219,9 @@ jar_app_layer = rule(
         "directory": attr.string(default = "/app"),
         # https://github.com/bazelbuild/bazel/issues/2176
         "data_path": attr.string(default = "."),
+        "workdir": attr.string(default = ""),
         "legacy_run_behavior": attr.bool(default = False),
+        "data": attr.label_list(cfg = "data", allow_files = True),
```

Comment:
should be host/target https://docs.bazel.build/versions/master/skylark/lib/attr.html#label_list